### ISSUE TITLE
official debian package?

### ISSUE THREAD

**Issue by anarcat** [rfrail3/tuptime#25]
do you still plan on taking on the official debian packaging? or should i package 3.2.3 for you in debian now?

**Comment by rfrail3** [comment]
Hi Anarcat,

Please, do a package for 3.2.3 in Debian now.

Thanks,

**Comment by anarcat** [comment]
i am sorry, i still haven't found the time to do this. if you want to build a new package, i'd be happy to sponsor it, but in the meantime, i can't find the time to do this just yet.

**Comment by rfrail3** [comment]
Ok, I will build the new one.

Thanks!

**Comment by rfrail3** [comment]
Note: This is delayed because I was adding more features to Tuptime and I prefer prepare the new build with the new version.

**Comment by rfrail3** [comment]
Hi Anarcat,

Please, continue packagin the project. I just did a new release with the 3.3.0 tag, I hope that you can build the .deb without issue.

Thanks,

**Comment by anarcat** [comment]
Hi Ricardo!

You marked yourself as maintainer in ef47227, yet used my name in `debian/changelog` - which means I cannot upload the package directly without marking it as a "Non maintainer upload". This is rather unusal: you should have used the same identity in the two locations. I will need to patch the changelog to fix that, which will make it diverge from you, something which we want to avoid!

I'll be happy to sponsor the upload once this is fixed.

**Comment by anarcat** [comment]
Another issue is that the package is marked as `3.0 (quilt)` format (ie. non-native) yet the version number (`3.3.0`) is native. You need to add a dash (e.g. `3.3.0-1`) to make it non-native or revert to the `3.0 (native)` format.

**Comment by anarcat** [comment]
those issues are somewhat fixed in #27.

**Comment by anarcat** [comment]
the last remaining issue is:

```
W: tuptime source: out-of-date-standards-version 3.9.6 (current is 3.9.7)
```

You can review what that change implies here:

https://www.debian.org/doc/debian-policy/upgrading-checklist

**Comment by rfrail3** [comment]
I just update the debian/control Standards-Version to the last 3.9.8.0

**Comment by anarcat** [comment]
Alright, thanks. So I rebuild the package with `git-buildpackage` in a `sid` environment. I use pbuilder, so the commandline was actually:

```
DIST=sid ARCH=amd64 git-buildpackage --git-tag
```

See https://wiki.debian.org/cowbuilder#Using_with_git-buildpackage for the configuration i am currently using.

I also used the tarball available in https://github.com/rfrail3/tuptime/releases/tag/3.3.0 to get a build consistent with upstream. Speaking of which, it would be nice to be able to trust this tarball in some way: if you would sign your tarballs with GPG, then they could be verified more easily (and even built in to the debian package infrastructure, through the uscan watch file).

Finally, I have made a small fix to the gbp.conf file to lay down the right tag (and pushed that tag to my personnal repo), it would be great if you would pull that tag!

The package has been uploaded to sid. Let me know when you make a new release!
